hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -147,6 +147,7 @@ enum Type {
   ListStatusLight = 129;
   GetSnapshotInfo = 130;
   RenameSnapshot = 131;
+  RefetchNetworkTopologyTree = 132;
```

Comment:
Let's omit `Tree` to make it a bit shorter.  (Also in request/response, method and command names.)